pkg/flow/logs.go

```diff
@@ -308,3 +346,39 @@ func (srv *server) logToMirrorActivity(ctx context.Context, t time.Time, act *en
 	srv.pubsub.NotifyMirrorActivityLogs(act)
 
 }
+
+func (srv *server) storeLogMsg(l *logMessage) error {
+	util.Trace(l.ctx, l.msg)
+	ctx := context.Background()
+	tx, err := srv.db.Tx(ctx)
```

Comment:
You need to rollback the transaction if this function fails for any reason. Usually we defer rollback because the library is smart enough to ignore a rollback if the commit has already happened.

Reply:
Looks like this has been resolved.